### PR TITLE
script to migrate remaining apps to vellum case management

### PR DESCRIPTION
**Pull request by orangejenny**
@emord / @czue

### CODE REVIEW

File: corehq/apps/app_manager/management/commands/migrate_all_apps_to_cmitfb.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import logging
+from django.core.management import BaseCommand, call_command
+
+from corehq.apps.app_manager.dbaccessors import get_app
```
Comment (dimagimon) [inline corehq/apps/app_manager/management/commands/migrate_all_apps_to_cmitfb.py]: F401 'get_app' imported but unused

File: corehq/apps/app_manager/management/commands/migrate_all_apps_to_cmitfb.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+class Command(BaseCommand):
+    help = '''
+        Migrate any non-migrated apps
+    '''
+
+    def handle(self, *args, **options):
+        # Find all apps not yet using vellum case management
+        app_query = AppES().is_build(False).term('vellum_case_management', False) \
+                           .term('doc_type', 'Application').source(['domain', '_id'])
```
Comment (emord) [inline corehq/apps/app_manager/management/commands/migrate_all_apps_to_cmitfb.py]: I don't think that you need `domain` and you could also use `get_ids()`  to get a list of ids

**Review by emord — APPROVED** [review]
Looks good once tests pass. My only comment isn't blocking

**Review by emord — APPROVED** [review]

**Comment by czue** [comment]
is there a timeline for running this?

**Comment by orangejenny** [comment]
@czue soon, likely today or tomorrow

**Comment by czue** [comment]
exciting! trust you all have done a good job thinking it through so it's smooth

**Comment by orangejenny** [comment]
We've indeed talked about it quite a bit; the number of domains is intimidating (19K). Will be watching the script run and paying attention to HQ performance.

**Comment by millerdev** [comment]
Awesome! 🎉 

Surely there is something we can kill as a token of the sacrifices we've made in this long and drawn-out migration process? I'm excited to see what comes next after this is done.

**Pull request merged**